src/modules/reading-room/components/GenreSelect/GenreSelect.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReactSelect, ReactSelectProps } from '@meemoo/react-components';
+import { FC } from 'react';
+import { useSelector } from 'react-redux';
+
+import { selectMediaResults } from '@shared/store/media';
+
+const GenreSelect: FC<ReactSelectProps> = (props) => {
+	const aggregates = useSelector(selectMediaResults)?.aggregations.schema_genre?.buckets;
+
+	const options = (aggregates || []).map((bucket) => ({
+		// label: `${bucket.key} (${bucket.doc_count})`, // Disabled due to non-representative scale of results
```

Comment:
Including this will open up the discussions what that number means, we can't supply both the full list of aggregates and the count of how many results there theoretically are when adding a certain filter without firing a million requests onto Elastic.